web-app/js/decode.js

```diff
@@ -0,0 +1,5 @@
+// https://stackoverflow.com/a/34064434
+function htmlDecode(input) {
+  const doc = new DOMParser().parseFromString(input, 'text/html');
+  return doc.documentElement.textContent;
+}
```

Comment:
I wanted to put this utils function inside of the `custom.gsp` layout meta tag but it appears to be that this
```
<meta name="layout" content="custom" />
```
is being executed after all of the scripts which resulted in a case where I wanted to use `htmlDecode` and it threw _"not defined error"_ so I decided to just import this script directly before the script that I will be using this `htmlDecode(...)`